composer.json

```diff
@@ -0,0 +1,31 @@
+{
+
+    "name": "oat/taoSyncServer",
```

Comment:
Composer package should be as repository name on GitHub - https://github.com/oat-sa/extension-tao-sync/blob/master/composer.json#L2

Reply:
fixed